browser/js/content.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+function init() {
+  // Only do work on reddit.
+  if (!location.hostname.endsWith('.reddit.com')) return;
```

Comment:
Is there any situation where the hostname is exactly `reddit.com`, which would break this? Or can we assume they'll always redirect the browser to `www.reddit.com` or similar?